atomist/get-started.md

```diff
@@ -0,0 +1,381 @@
+    - `SERVICE_ACCOUNT_EMAIL`: the service account address; a combination of the
+      service account ID and project ID.
+    - `SUBSCRIPTION`: the name of the PubSub to be created (can be anything).
+
+    ```bash
+    PUSH_ENDPOINT_URI={COPY_THIS_FROM_ATOMIST}
+    SERVICE_ACCOUNT_EMAIL="${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com"
+    SUBSCRIPTION="atomist-gar-integration-subscription"
+    ```
+
+11. Create the PubSub for the `gcr` topic.
+
+    ```bash
+    gcloud pubsub subscriptions create ${SUBSCRIPTION} \
+      --topic='gcr' \
+      --push-auth-token-audience='atomist' \
+      --push-auth-service-account="${SERVICE_ACCOUNT_EMAIL}" \
+      --push-endpoint="${PUSH_ENDPOINT_URI}"
+    ```
+
+When the first image push is successfully detected, a green check mark on the
+integration page will indicate that the webhook event was received and that the
+integration works.
+
+</div>
+<div id="tab-ghcr" class="tab-pane fade" markdown="1">
+<!-- GitHub Container Registry -->
+
+To integrate Atomist with GitHub Container Registry, connect your GitHub
+account, and enter a personal access token for Atomist to use when pulling
+container images.
+
+1. Go to <https://dso.docker.com> and sign in using your Docker ID credentials.
+2. Connect your GitHub account as instructed in the
+   [GitHub app page](./integrate/github.md#connect-to-github). Install the app
+   into the GitHub account that contains your GitHub Container Registry.
+3. Open the [**Integrations**](https://dso.docker.com/r/auth/integrations){:
+   target="blank" rel="noopener" class=""} tab, and click the **Configure** link
+   next to the **GitHub Container Registry** in the list of integrations.
+4. Fill out the fields and click **Save Configuration**.
+
+   The **Personal access token** is required for scanning images in private
+   repositories. The token must have the
+   [`read:packages` scope](https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages).
+
+   You can leave the **Personal access token** field blank if you only want to
+   scan images in public repositories.
+
+</div>
+<hr>
+</div>
+
+## Link images to Git repository
+
+Knowing the source repository of an image is a prerequisite for Atomist to
+interact with the Git repository, such as adding scan results as a status check
+in pull requests. For Atomist to be able to link scanned images to a Git
+repository repository, the image must be annotated with labels at build time.
+
+The image labels that Atomist requires are:
+
+| Label                                | Value                                             |
+| ------------------------------------ | ------------------------------------------------- |
+| `org.opencontainers.image.revision`  | The commit revision that the image is built for.  |
+| `org.opencountainers.image.source`   | HTTP(S) address of the project.                   |
```

Comment:
Typo

```suggestion
| `org.opencontainers.image.source`   | HTTP(S) address of the project.                   |
```
About description, we should keep the same as OCI ones: https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

```
| `org.opencontainers.image.revision` | Source control revision identifier for the packaged software. |
| `org.opencontainers.image.source`   | URL to get source code for building the image.                   |
```

And maybe also add a link to the specs so users are not confused: https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys